fastlane-plugin-wpmreleasetoolkit.gemspec

```diff
@@ -37,6 +37,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'progress_bar', '~> 1.3'
   spec.add_dependency 'rake', '>= 12.3', '< 14.0'
   spec.add_dependency 'rake-compiler', '~> 1.0'
+  spec.add_dependency 'xcodeproj', '~> 1.22.0'
```

Comment:
This is too specific of a constraint.

That `~>` means that it will only allow to blindly update the last component of the version, so `~>1.22.0` will only allow us to update to any `1.22.z` with `z>=0`. But semantic versioning makes it ok to also get any new minor version — so as long as it's still a major version 1 (`1.y.z`) we should not refrain xcodeproj from being updated.

This is particularly important to have a more relaxed constraint here because `xcodeproj` is also used by other gems of our client apps' dependency tree, especially its a dependency of both `cocoapods` and `fastlane`, and we don't want the release-toolkit to generate a dependency conflict and prevent `xcodeproj` to be updated to a newer non-breaking / minor version is one of those gems require it in the future.

Reply:
True, minor updates should be ok in general 👍 besides that, I didn't realise the issue with client apps' dependency tree (to be fair, even release-toolkit own's dependency tree I guess) in this case, good point.

Reply:
Done in 70e3dc38421823ee71c7f7c55d53c734b7b4bfc7